README.md

```diff
@@ -211,12 +211,12 @@ These APIs are triggered by an entity (Invoker or Exposer for release 1.0) to:
 #### Register an entity
 Request
 ```shell
-curl --request POST 'http://<CAPIF_HOSTNAME>/register' --header 'Content-Type: application/json' --data '{
+curl --request POST 'http://<CAPIF_HOSTNAME>:8080/register' --header 'Content-Type: application/json' --data '{
```

Comment:
not always will be port 8080, for example, at deployment on openshift the port is 80, this is why in previous version of curl command I didn't setup a port at request to http endpoint. Maybe we can put something of <CAPIF_HTTP_PORT>=:8080